update types

zeroxdead opened the pull request:
1. loadInterfaceXML, loadSubInterfaceXML are taken from https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/main/js/misc/dbusUtils.js?ref_type=heads
2. title, subtitle can be null according to https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/main/js/ui/quickSettings.js?ref_type=heads

Totto16 commented:
Thanks for the PR.

I'll look into it. But from the surface it seems correct, but could you add a ```@version 46``` doc comment to the changed functions (or all those, you looked into and where fine for gnome 46) like already present on some files (e.g. [here](https://github.com/gjsify/gnome-shell/blob/main/packages/gnome-shell/src/ui/dialog.d.ts#L6-L9))

zeroxdead commented:
1. Removed loadSubInterfaceXML from fileUtils.d.ts since it is not actually used in fileUtils.js in any way
2. Changed loadInterfaceXML declaration in fileUtils.d.ts to a re-export from dbusUtils.js as it is done in fileUtils.js
3. Added links + version annotation as you asked

Totto16 reviewed (approved):
LGTM ❤️
